W-20072381-App-upgrade-tool-Eligibility-LDS (#827)
mulesoft/docs-runtime-manager: cloudhub/modules/ROOT/pages/app-migration.adoc

[NOTE]
During the application upgrade process, MuleSoft doesn't provide any out-of-the-box entitlements. Work with your account representative to remain compliant with entitlements and finish the upgrade on time.

== Eligibility

> To link a CloudHub application with a CloudHub 2.0 application and gradually switch traffic using the application migration tool, you must first upgrade the CloudHub VPC to a CloudHub 2.0 private space using the xref:vpc-upgrade.adoc[VPC upgrade tool].

Reviewer: ```suggestion
To link a CloudHub application with a CloudHub 2.0 application and gradually switch traffic using the application migration tool, first upgrade the CloudHub VPC to a CloudHub 2.0 private space by using the xref:vpc-upgrade.adoc[VPC upgrade tool].
```

> If you move applications into a CloudHub 2.0 private space that hasn't been upgraded from a CloudHub VPC, you can't use the application upgrade tool.

Reviewer: If you want to use a positive sentence (unless it changes the meaning): 

"You can use the application upgrade tool only in CloudHub 2.0 private spaces that were upgraded from a CloudHub VPC."
